argilla-server/src/argilla_server/apis/v1/handlers/metadata_properties.py

```diff
@@ -88,11 +88,4 @@ async def delete_metadata_property(
 
     await authorize(current_user, MetadataPropertyPolicyV1.delete(metadata_property))
 
-    # TODO: We should split API v1 into different FastAPI apps so we can customize error management.
-    # After mapping ValueError to 422 errors for API v1 then we can remove this try except.
-    try:
-        await datasets.delete_metadata_property(db, metadata_property)
-    except ValueError as err:
-        raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_ENTITY, detail=str(err))
-
-    return metadata_property
+    return await datasets.delete_metadata_property(db, metadata_property)
```

Comment:
Do we have any test checking this change? The context function is not returning UnprocessableEntityError and the ValueError is not handled by the error handler

Reply:
There is no need to add tests because the context function can't raise any `ValueError`. It's only doing a database query to delete the record.

I believe the `except ValueError as err` capture statement was added following the style of the rest of handlers without noticing the code is never raising `ValueError`.